src/pages/services/aem-universal-editor/api/item-types-renderers/index.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Custom item types renderers for properties rail - Universal Editor Extensibility
+description: Discover how to personalize the user interface of a data type field within the properties rail of Universal Editor.
+contributors:
+  - https://github.com/AdobeDocs/uix
+---
+
+# Custom element rendering for properties rail by data-type
+
+This feature allows third-party developer to build custom input UI for specific data types in the Universal Editor. 
+The custom UI is rendered in an iframe and replaces the standard UI for the field in the [properties rail](https://experienceleague.adobe.com/en/docs/experience-manager-cloud-service/content/sites/authoring/universal-editor/authoring#properties-rail).
+
+![](./custom-renderer-field.png)
+
+## Define override rules
+
+An UIX extension can define a custom renderer to replace the standard UI  with an iframe, which then renders the custom UI provided by the extension.
+Check this document to learn about available [data types for universal editor](https://experienceleague.adobe.com/en/docs/experience-manager-cloud-service/content/implementing/developing/universal-editor/attributes-types#item-types). 
+```js
+function ExtensionRegistration() {
+  const init = async () => {
+    const guestConnection = await register({
+      id: extensionId,
+      methods: {
+        canvas: {
+          getRenderers() {
+            return [
+              {
+                dataType: "text",
+                url: '/#/renderer/1'
+              },
+              {
+                dataType: "custom-type",
+                url: '/#/renderer/2',
+              },
+            ];
+          },
+        },
+      },
+    });
+  };
+  init().catch(console.error);
+
+  return <Text>IFrame for integration with Host (AEM)...</Text>;
+}
+```
+
+## Implement custom rendering logic
+
+1. Define the router in the extension application to handle the custom rendering of the field.
+```js
+function App() {
+  return (
+    <Router>
+      <ErrorBoundary onError={onError} FallbackComponent={fallbackComponent}>
+        <Routes>
+            <Route index element={<ExtensionRegistration />} />
+            <Route
+                exact
+                path="index.html"
+                element={<ExtensionRegistration />}
+            />
+            <Route
+                exact path="renderer/:rendererId"
+                element={<TypeRenderer />}
+            />
+        </Routes>
+      </ErrorBoundary>
+    </Router>
+  )
+
+  // Methods
+
+  // error handler on UI rendering failure
+  function onError(e, componentStack) {}
+
+  // component to show if UI fails rendering
+  function fallbackComponent({ componentStack, error }) {
+    ...
+}
+
+export default App
+```
+
+2. Implement custom field renderer
+
+```js
+export default () => {
+    const [isLoaded, setIsLoaded] = useState(true);
```

Comment:
Not a big issue, but I think it supposed to be `isLoading`

Reply:
fixed